test/blocks/merch-card/merch-card.test.js

```diff
@@ -60,6 +60,21 @@ describe('Merch Card', () => {
     expect(buttons[1].textContent).to.be.equal('Save now');
   });
 
+  it('Supports Card as a Fragment', async () => {
+    document.body.innerHTML = await readFile({ path: './mocks/special-offers.html' });
+    await init(document.querySelector('.special-offers-black-friday'));
+    const fragment = document.querySelector('.special-offers-fragment');
+    const fragmentSection = fragment.querySelector('.section');
+    const card = fragment.querySelector('.merch-card');
+
+    expect(fragment).to.be.exist;
+    expect(fragment.parentElement.children.length).to.be.equal(3);
```

Comment:
You could also use `childElementCount` on the parent. And I think `.be` can be removed (valid for the next few checks as well)